bundles/org.openhab.binding.bluelink/README.md

```diff
@@ -0,0 +1,222 @@
+| `seat-rear-left`     | Number             | R          | Rear left seat heater (0-3)  |
+| `seat-rear-right`    | Number             | R          | Rear right seat heater (0-3) |
+| `steering-heater`    | Switch             | R          | Steering wheel heater        |
+| `rear-window-heater` | Switch             | R          | Rear window heater           |
+| `side-mirror-heater` | Switch             | R          | Side mirror heater           |
+
+### Range Group
+
+| Channel       | Type          | Read/Write | Description             |
+|---------------|---------------|------------|-------------------------|
+| `total-range` | Number:Length | R          | Total driving range     |
+| `ev-range`    | Number:Length | R          | EV-only driving range   |
+| `fuel-range`  | Number:Length | R          | Fuel-only driving range |
+
+### Fuel Group (ICE vehicles only)
+
+This channel group is only available for internal combustion engine (ICE) vehicles.
+
+| Channel       | Type                 | Read/Write | Description         |
+|---------------|----------------------|------------|---------------------|
+| `level`       | Number:Dimensionless | R          | Fuel tank level (%) |
+| `low-warning` | Switch               | R          | Low fuel warning    |
+
+### Charging Group (Electric/Hybrid vehicles only)
+
+This channel group is only available for electric and hybrid vehicles.
+
+| Channel                 | Type                 | Read/Write | Description                     |
+|-------------------------|----------------------|------------|---------------------------------|
+| `soc`                   | Number:Dimensionless | R          | Battery state of charge (%)     |
+| `charging`              | Switch               | RW         | Charging status (ON to start)   |
+| `plugged-in`            | Switch               | R          | Charge cable plugged in         |
+| `charge-limit-dc`       | Number:Dimensionless | RW         | DC charge limit (%)             |
+| `charge-limit-ac`       | Number:Dimensionless | RW         | AC charge limit (%)             |
+| `time-to-full-current`  | Number:Time          | R          | Time to full (current charger)  |
+| `time-to-full-fast`     | Number:Time          | R          | Time to full (fast charger)     |
+| `time-to-full-portable` | Number:Time          | R          | Time to full (portable charger) |
+| `time-to-full-station`  | Number:Time          | R          | Time to full (station charger)  |
+
+### Warnings Group
+
+| Channel            | Type   | Read/Write | Description |
+|--------------------|--------|------------|-------------|
+| `tire-pressure`    | Switch | R          | Tire pressure warning |
+| `washer-fluid`     | Switch | R          | Washer fluid low warning |
+| `brake-fluid`      | Switch | R          | Brake fluid warning |
+| `smart-key-battery`| Switch | R          | Smart key battery warning |
+
+## Full Example
+
+### things/bluelink.things
+
+```
```

Comment:
```suggestion
```java
```

Reply:
any particular reason for having `java` here? all it does in IntelliJ is add a bunch of red squiggly lines.